.registry/app.yaml

```diff
@@ -1,4 +1,4 @@
-title: "Minimal GCP Environment"
+title: "Sample K8s and Managed Services"
```

Comment:
Are we sure we don't want to keep any reference to GCP here? Assuming we follow this pattern this name would "collide" with the other sample stacks, no?

Reply:
right, how about “GCP Sample Stack for Kubernetes and Managed Service”